dhctl/cmd/dhctl/commands/bootstrap/bootstrap.go

```diff
@@ -23,8 +23,8 @@ import (
 	"github.com/deckhouse/deckhouse/dhctl/pkg/terraform"
 )
 
-func DefineBootstrapCommand(kpApp *kingpin.Application) *kingpin.CmdClause {
-	cmd := kpApp.Command("bootstrap", "Bootstrap cluster.")
+func DefineBootstrapCommand(kpApp *kingpin.Application, command string) *kingpin.CmdClause {
```

Comment:
let move not only command string and all 
`kpApp.Command(command, "Bootstrap cluster.")` to up level

I think it will be more cleared

Reply:
Done